boards/xtensa/esp32/esp32.dts

```diff
@@ -24,11 +24,11 @@
 };
 
 &cpu0 {
-	clock-frequency = <ESP32_CLK_CPU_80M>;
+	clock-frequency = <ESP32_CLK_CPU_240M>;
```

Comment:
But wasn't this "bug" introduced in the first commit?  Fix it there, please.

Reply:
What do you mean by fix it here?

Once bootloader and XIP was taken in place on 1st commit, system clock was not working for frequencies > 80MHz. That is the reason that on 1st commit it was changed to 80MHz as default.
This commit fixes it and reset default clock rate to 240MHz.

Reply:
> Yes, you have a point. I'll fix in there, keep 240MHz as default.

Well, I won't insist on that, if doing that breaks builds with the intervening commits (e.g. when doing git bisect).  An acceptable alternative is to document in the first commit why it's being changed to 80 MHz, noting that that will be fixed after other changes occur.

But if you can avoid the breakage by refactoring a little that'd be fine too.

Reply:
I squashed both commits into the 1st one. It means 1st commit regarding bootloader and XIP stuff will have also clock source changed. Sounds good?

Reply:
I'll have to see it (and how the commit comment is updated), but if the devicetree content doesn't change I wouldn't notice there was a problem, so it's probably fine.

Reply:
There are a few changes on 1st commit regarding code style.. Can we keep it?

Reply:
It'd certainly be preferable to not mix whitespace and line wrapping changes in with things that have an actual effect on behavior.  I'm not sure I'd insist on it, but others would, so it could be worth dropping the changes that are cosmetic-only.

Reply:
Removed all comestic stuff.